pkg/cmd/playback.go

```diff
@@ -190,24 +194,79 @@ func (pc *playbackCmd) runPlaybackCmd(cmd *cobra.Command, args []string) error {
 	select {}
 }
 
-func startListenCmdLoop(mode string, address string, httpWrapper *playback.Server) {
-	lc := newListenCmd()
-	lc.forwardURL = address + "/playback/webhooks"
-	startListenCmd := func() {
+func runListen(cmd *cobra.Command, address string) error {
+	version.CheckLatestVersion()
+
+	deviceName, err := Config.Profile.GetDeviceName()
+	if err != nil {
+		return err
+	}
+
+	key, err := Config.Profile.GetAPIKey(false)
+	if err != nil {
+		return err
+	}
+
+	ctx := withSIGTERMCancel(cmd.Context(), func() {
+		log.WithFields(log.Fields{
+			"prefix": "proxy.Proxy.Run",
```

Comment:
we prob want to change that, to add context on the fact that this is proxy ran from playback.

"playback.proxy.Run" might be ok.

Reply:
sounds good I'll switch it to that